chromium_src/content/browser/utility_process_host.cc

```diff
@@ -0,0 +1,14 @@
+/* Copyright 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#define BRAVE_UTILITYPROCESSHOST_STARTPROCESS                              \
+  const bool is_service = service_identity_.has_value();                   \
+  if (is_service) {                                                        \
+    GetContentClient()->browser()->AdjustUtilityServiceProcessCommandLine( \
```

Comment:
@darkdh why were we doing this in `AdjustUtilityServiceProcessCommandLine` instead of `AppendExtraCommandLineSwitches`? Seems like the latter would work fine for this

Reply:
`AppendExtraCommandLineSwitches` is for renderer not utility process

Reply:
I was wrong, we can move the append extra switches logic from `BraveContentBrowserClient::AdjustUtilityServiceProcessCommandLine` to `AdjustUtilityServiceProcessCommandLine` and check `process_type == switches::kUtilityProcess`

Reply:
I will the do follow up here https://github.com/brave/brave-browser/issues/8761